### ISSUE TITLE
Cant create a group even as a global admin?

### ISSUE THREAD

**Issue by skyflocodes-simcoe** [OfficeDev/microsoft-teams-apps-groupconnect#26]
![image](https://user-images.githubusercontent.com/84514584/120232313-5292e500-c221-11eb-84b1-ecb0783e16a9.png)
red error message pops up for any team I try and I'm in my dev tenant.. I'm the global admin.

**Comment by himani-chi** [comment]
Are you a team owner of the team you're trying to create ERG group in?

**Comment by skyflocodes-simcoe** [comment]
> Are you a team owner of the team you're trying to create ERG group in?

Yes. I have tried to add it to 6+ teams all created and owned by me and this error shows up on all of them.
